specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/webapplicationfirewall.json

```diff
@@ -686,6 +694,80 @@
           "description": "When matchVariable is a collection, operator used to specify which elements in the collection this exclusion applies to."
         }
       }
+    },
+    "BotDetection": {
+      "description": "Allows the detection of bots and specify action for each bot category.",
+      "required": [
+        "good", 
```

Comment:
What action would we perform on the good bots? Do we really need them?

Reply:
i just saw it wasnt required for custom rules so i didnt think it was necessary but i see your point i agree. not sure how the custom rules work for logging. ill make it required to be safe

Reply:
again, this was a AFD's requirements